src/utils.ts

```diff
@@ -105,36 +105,36 @@ export function handleErrors(errors: string[]) {
 }
 
 export async function createPullSecret({
-  teamId,
+  team,
   name,
   server,
   password,
   username = '_json_key',
 }: {
-  teamId: string
+  team: string
   name: string
   server: string
   password: string
   username?: string
 }): Promise<void> {
   const client = getApiClient()
-  const namespace = `team-${teamId}`
+  const namespace = team
```

Comment:
this is incorrect, and not discussed refactoring that makes the code less clear....please revert to using `teamId` throughout and avoid using the variable name `team`.